dogs.html

```diff
@@ -27,88 +27,88 @@
         <strong>Jake</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
       </div>
       <div class="card">
-        <img src="images/angus-card.jpg" alt="Angus">
+        <img onclick="alert('Angus, Mutt, $123.45')" src="images/angus-card.jpg" alt="Angus">
         <strong>Angus</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
       </div>
       <div class="card">
-        <img src="images/violet-card.jpg" alt="Violet">
+        <img onclick="alert('Violet, Chocolate Lab, $123.45')" src="images/violet-card.jpg" alt="Violet">
         <strong>Violet</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
       </div>  
       <div class="card">
-        <img src="images/piper-card.jpg" alt="Piper">
+        <img onclick="alert('Piper, Mutt, $123.45')" src="images/piper-card.jpg" alt="Piper">
         <strong>Piper</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
       </div>
       <div class="card">
-        <img src="images/maximus-card.jpg" alt="Maximus">
+        <img onclick="alert('Maximus, Terrier, $123.45')" src="images/maximus-card.jpg" alt="Maximus">
         <strong>Maximus</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
       </div>
       <div class="card">
-        <img src="images/luna-card.jpg" alt="Luna">
+        <img onclick="alert('Luna, Lab, $123.45')" src="images/luna-card.jpg" alt="Luna">
         <strong>Luna</strong>
         <span>Cost to Adopt: $123.45</span>
         <p>Corrum volorit landae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis ra ea erum, qui doluptae</p>
-        <button>Adopt</button>
+        <button onclick="cartTotal()">Adopt</button>
       </div>
       <div class="card">
-        <img src="images/stella-card.jpg" alt="Stella">
+        <img onclick="alert('Stella, Chihuahua, $123.45')" src="images/stella-card.jpg" alt="Stella">
```

Comment:
Great. Totally does what is expected. As an improvement, see if you could move this logic to a function? It's kind of overkill, but a good habit to get into.

```
<img onclick="showDogInfo('Stella', 'Chihuahua', 123.45)" src="images/stella-card.jpg" alt="Stella">
```
```
function showDogInfo(name, breed, cost) {
  let dogString = name + ', ' + breed + ', $' + cost;
  alert(dogString);
}
```